### PR TITLE
Appc 978 create trivial drive sample to use bds billing sdk

### PR DESCRIPTION
**Pull request by brunovarelaAptoide**
**What does this PR do?**

Adapts  IABSample to the new Billing SDK.Removes the IABHelper and all old classes and uses the lib from Billing SDK.
Improvements choosing the link to DEV or PROD in ONE_STEP.

**Database changed?**

No

**Where should the reviewer start?**
- MainActivity.java

**How should this be manually tested?**

- Have the Appcoins Wallet DEV Wallet installed.
- Build with the Variant appcoinsiabRopsten.

**What are the relevant tickets?**

Tickets related to this pull-request: APPC-978

**Code Review Checklist**

[] Architecture
[] Documentation on public interfaces
[] Database changed?
[] If yes - Database migration?
[]  Remove comments & unused code & forgotten testing Logs
[]  Codestyle
[]  Functional QA tests pass

### CODE REVIEW

File: app/src/appcoinssdk/java/com/aptoide/iabexample/utilssdk/ApplicationUtils.java [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+        + "-1005:User cancelled/"
+        + "-1006:Unknown purchase response/"
+        + "-1007:Missing token/"
+        + "-1008:Unknown error/"
+        + "-1009:Subscriptions not available/"
+        + "-1010:Invalid consumption attempt").split("/");
+
+    if (code <= -1000) {
+      int index = -1000 - code;
+      if (index >= 0 && index < iabhelper_msgs.length) {
```
Comment (JDSM01) [inline app/src/appcoinssdk/java/com/aptoide/iabexample/utilssdk/ApplicationUtils.java]: the index >=0 condition is always true due to the if(code <= -1000) condition. so remove one or another

File: app/src/appcoinssdk/java/com/aptoide/iabexample/utilssdk/PurchaseFinishedListener.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package com.aptoide.iabexample.utilssdk;
+
+public interface PurchaseFinishedListener {
+  void onPurchaseFinished(int responseCode,String message,String token, String sku);
+}
```
Comment (JDSM01) [inline app/src/appcoinssdk/java/com/aptoide/iabexample/utilssdk/PurchaseFinishedListener.java]: codestyle

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -13,12 +13,12 @@
       android:theme="@style/AppTheme">
     <activity
         android:name=".MainActivity"
-        android:screenOrientation="landscape">
+        android:screenOrientation="sensor">
```
Comment (JDSM01) [inline app/src/main/AndroidManifest.xml]: Question: What’s the intention behind this?

Comment (brunovarelaAptoide) [reply app/src/main/AndroidManifest.xml]: Use the trivial drive application in Portrait.

**Pull request merged**